duktape/build.gradle

```diff
@@ -1,124 +1,56 @@
 buildscript {
   dependencies {
-    classpath 'com.android.tools.build:gradle-experimental:0.7.0'
+    classpath 'com.android.tools.build:gradle:2.2.2'
   }
 }
 
-apply plugin: 'com.android.model.library'
+apply plugin: 'com.android.library'
 
-model {
-  android {
-    compileSdkVersion = rootProject.ext.compileSdkVersion
-    buildToolsVersion = rootProject.ext.buildToolsVersion
-
-    defaultConfig {
-      minSdkVersion.apiLevel = 4
+android {
+  compileSdkVersion 25
+  buildToolsVersion '23.0.3'
+  defaultConfig {
+    minSdkVersion 15
```

Comment:
Turns out this was needed for the native part of the build?  I can go as low as 12 and have it still work.